### ISSUE TITLE
Integration doesn't respond with OK!

### ISSUE THREAD

**Issue by koostamas** [junalmeida/homeassistant-torque#29]
## Version of the custom_component
Unpublished

## Describe the bug
I am trying to use this integration, but I found that the endpoint /api/torque_logger doesn't respond with the message "OK!" like the built-in integration, but with a generic HTML page with just the Home Assistant logo in the middle, so Torque doesn't recognise the messages as being sent successfully and tries to send all of them repeatedly. The integration also doesn't create any sensors, so it might not even receive the data. There aren't any related log entries. I am using HA docker 2024.8.3.
Also, I noticed that the endpoint doesn't throw a 401 code even if I don't send a bearer token, so this leads me to beliveve that something is just not set up correctly on my side (but it does do something, because without the integration installed, I obviously get a 404 for the endpoint).

Any ideas what the issue might be?

Update: I guess the get callback is not getting called alltogether, and I also can't see anything in the logs even if I enable debug logging for the integration.

**Comment by junalmeida** [comment]
Have you tested on a supervised HA instance?

**Comment by koostamas** [comment]
No, but I actually figured out what the issue was. I am using Nginx as reverse proxy in front of HA and require client certificates there. Since the Torque app can't send client certs, I created a separate location for the integration api that didn't require client certificates, but I messed up the `proxy_pass` url, so it was returning the HA homepage HTML and not passing the request onto the integration.
